projects/ng-components-docs/src/app/examples/components/pagination/examples/pagination/pagination.component.html

```diff
@@ -0,0 +1,8 @@
+<sebng-pagination
+    [value]="pagination"
+    (onChange)="setPagination($event)"
```

Comment:
Angular doesn't like to use `on` prefix on methods and in a lot of cases, it will throw an error. Can you replace it with just `change`